src/main/java/gregtech/loaders/recipe/chemistry/SeparationRecipes.java

```diff
@@ -566,4 +567,27 @@ public static void init() {
                 .duration(300).EUt(2).buildAndRegister();
 
     }
+
+    @SuppressWarnings("unchecked")
+    private static List<ItemStack> getGrassSeedItems() {
+        List<ItemStack> result = new ArrayList<>();
+        try {
+            Field seedListField = ForgeHooks.class.getDeclaredField("seedList");
+            seedListField.setAccessible(true);
+            Class<?> seedEntryClass = Class.forName("net.minecraftforge.common.ForgeHooks$SeedEntry");
+            Field seedField = seedEntryClass.getDeclaredField("seed");
+            seedField.setAccessible(true);
+
+            List<WeightedRandom.Item> seedList = (List<WeightedRandom.Item>) seedListField.get(null);
+            for (WeightedRandom.Item seedEntryObject : seedList) {
+                ItemStack seedStack = (ItemStack) seedField.get(seedEntryObject);
+                if (!seedStack.isEmpty()) {
+                    result.add(seedStack);
+                }
+            }
+        } catch (ReflectiveOperationException exception) {
```

Comment:
This catch block can be made more specific:
```suggestion
        } catch (NoSuchFieldException | IllegalAccessException | ClassNotFoundException e) {
```